server/src/main/java/io/crate/role/Role.java

```diff
@@ -49,12 +48,11 @@ public enum UserRole {
     private final SecureHash password;
     private final Set<UserRole> userRoles;
 
-    @VisibleForTesting
-    protected Role(String name,
-                   boolean isUser,
-                   Set<Privilege> privileges,
-                   @Nullable SecureHash password,
-                   Set<UserRole> userRoles) {
+    public Role(String name,
+                boolean isUser,
+                Set<Privilege> privileges,
+                @Nullable SecureHash password,
+                Set<UserRole> userRoles) {
```

Comment:
How about just one overload of the ctor without this last arg which is only used for the CRATE superuser?

Reply:
One different usage doesn't justify an overload in my opinion.

Reply:
totally fine, thx!